photos/photo.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
this JSON data is duplicated in another component. let's consolidate it and export it from here. 
nit: the folder name should be utils or data

Reply:
i removed the JSON data since its only a dummy data